### DISCUSSION TITLE
Resources 폴더 내부 경로를 추가 설정시 접근 불가

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#146] [Issue Report (Legacy)]
Resources 폴더에 리소스를 넣고 로드하면 로드가 가능하나
폴더를 하나 만들고 폴더 안 파일을 로드하면 안됩니다.
에디터에서는 로드가 되는데 QR 찍고 실행하면 로드가 안됩니다.

![image](https://user-images.githubusercontent.com/25768804/147452802-dc01649a-ec47-43d1-bb62-7e88d6bb85a0.png)
 - 로드됨

![image](https://user-images.githubusercontent.com/25768804/147452845-04cf83a2-bf1f-4405-a963-70140c10a6d9.png)
 - 로드안됨

**Comment by JeongDongBaek** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/209#discussion-3797504
같은 이슈입니다.
